tensorflow/contrib/ios_examples/camera/CameraExampleViewController.mm

```diff
@@ -22,6 +22,27 @@
 
 #include "tensorflow_utils.h"
 
+// If you have your own model, modify this to the file name, and make sure
+// you've added the file to your app resources too.
+static NSString* model_file_name = @"tensorflow_inception_graph";
+static NSString* model_file_type = @"pb";
+// This controls whether we'll be loading a plain GraphDef proto, or a
+// file created by the convert_graphdef_memmapped_format utility that wraps a
+// GraphDef and parameter file that can be mapped into memory from file to
+// reduce overall memory usage.
+const bool model_uses_memory_mapping = false;
```

Comment:
should this and the ones further below (wanted_input_* through output_layer_name) be static too?